modules/ROOT/pages/basic-authentication-simple-concept.adoc

```diff
@@ -5,29 +5,34 @@ endif::[]
 
 [width="100%", cols="5,15"]
 |===
->s| Policy Name | Basic Authentication: Simple
+>s| Policy Name | Basic Authentication - Simple
 >s|Summary      | Allows access based on the basic authorization mechanism, with a single user-password
 >s|Category |
->s| Since Mule Version | 4.0.0
+>s| Available from Mule version | 4.0.0
 .1+>.^s| Returned Status Codes
-|
+|401 - Unauthorized or invalid client application credentials
```

Comment:
Is the hyphen actually part of the display?

Reply:
I don't think so.